package.json

```diff
@@ -75,11 +77,12 @@
     "@gnosis.pm/owl-token": "^3.1.0",
     "@gnosis.pm/solidity-data-structures": "^1.2.4",
     "@gnosis.pm/util-contracts": "^2.0.6",
+    "@openeth/truffle-typings": "^0.0.1",
     "@truffle/contract": "^4.1.14",
     "@types/bn.js": "^4.11.6",
     "@types/chai": "^4.2.11",
-    "@types/mocha": "^7.0.2",
     "axios": "^0.19.2",
+    "bignumber.js": "^9.0.0",
```

Comment:
Is this needed for the generated types?

Reply:
yes. I believe either web3 or truffle was importing it so we needed the types which are part of the main package.